erp/imports/mapper/typeform.py

```diff
@@ -32,7 +32,11 @@ def csv_to_erp(self, record):
                 dest_fields["accessibilite"]["entree_plain_pied"] = False
 
             field_label = "Combien de marches y a-t-il pour entrer dans votre mairie ?"
-            dest_fields["accessibilite"]["entree_marches"] = record[field_label]
+            try:
+                nb_marches = int(record[field_label])
+            except ValueError:
```

Comment:
if `record[field_label]` can be `None` I would except `ValueError` and `TypeError`